etc/adminhtml/system.xml

```diff
@@ -226,6 +226,17 @@
                     </comment>
                     <source_model>Nosto\Tagging\Model\Config\Source\MultiCurrency</source_model>
                 </field>
+                <field id="pricing_variation" translate="label comment" type="select" sortOrder="11"
+                       showInDefault="1" showInWebsite="1" showInStore="1">
+                    <depends>
+                        <field id="method">disabled</field>
+                    </depends>
+                    <label>Price Variation</label>
+                    <comment><![CDATA[Enable price variation to support group price and catalog price rules.
```

Comment:
Let's use the same message as in M1. Copy Paste it.

Reply:
@mridang Do you mean this one?
https://github.com/Nosto/nosto-magento/blob/beta/app/code/community/Nosto/Tagging/etc/system.xml#L372